apps/api/src/test-data/utilities/convertInMemorySnapshotToDatabaseFormat.ts

```diff
@@ -12,11 +12,13 @@ import buildEdgeDocumentsFromCategoryNodeDTOs from '../../persistence/database/u
 import mapCategoryDTOToArangoDocument from '../../persistence/database/utilities/category/mapCategoryDTOToArangoDocument';
 import mapEdgeConnectionDTOToArangoEdgeDocument from '../../persistence/database/utilities/mapEdgeConnectionDTOToArangoEdgeDocument';
 import mapEntityDTOToDatabaseDTO from '../../persistence/database/utilities/mapEntityDTOToDatabaseDTO';
+import { ArangoMigrationRecord } from '../../persistence/migrations/migration-record';
+import { DTO } from '../../types/DTO';
 
-type InMemoryDatabaseSnapshot = {
+export type InMemoryDatabaseSnapshot = {
```

Comment:
I get nervous about maintaining this second mapping layer from domain to persistence. Maybe the right approach is to use the repositories to create the state in the database and then use the `ArangoDataExporter` to dump the data.

Reply:
[Here](https://www.pivotaltracker.com/story/show/185212566) is that story.